ui/src/common/components/ExpandListToggle.jsx

```diff
@@ -5,13 +5,13 @@ import SecondaryButton from './SecondaryButton';
 
 class ExpandListToggle extends Component {
   render() {
-    const { size, limit, onToggle, expanded } = this.props;
+    const { size, limit, onToggle, expanded, toggleLabel } = this.props;
```

Comment:
`expandLabel` would be better since it's only used to expand, otherwise `Hide` is rendered.

Ideally `renderLabel(expanded)` prop would be more flexible.